waku/wdd.md

```diff
@@ -0,0 +1,75 @@
+# Waku DNS Discovery
+
+> Version: 0.1.0 (Draft)
+>
+> Authors: Dean Eigenmann <dean@status.im>
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Specification](#specification)
+    1. [Records](#records)
+    2. [Traversing](#traversing)
+3. [Footnotes](#footnotes)
+4. [Copyright](#copyright)
+
+## Abstract
+
+In this specification, we describe the DNS discovery protocol used by [Waku](./waku.md). 
```

Comment:
And what specific use case does this have? E.g. in EIP it is for bootstrap nodes (mostly), vs Discovery v5 for regular nodes

I'd also like to see rationale for why not Discovery v5, and when you might want to use it (further peer discovery, etc)

Reply:
If we also intend to use it for regular nodes (at least initially), then do we have any ideas on how to solve the

"find closest mailserver/waku node" problem to avoid disconnects due to timeouts etc? If not, do we consider this out of scope? E.g. first get peer candidate list, then do pings to find closest node.

Reply:
@oskarth I think we consider this out of scope for this and expect nodes to do this themselves.